dev.json

```diff
@@ -2,7 +2,7 @@
    "base_url": "https://addons-dev.allizom.org",
    "detail_extension_slug": "awesome-screenshot-plus-",
    "duplicate_guid": "@contain-facebook",
-   "approved_addon_with_sources": "approved-src-code",
+   "approved_addon_with_sources": "jswszwpeoc",
```

Comment:
what does this change do?

Reply:
I've changed the variable for the addon used in the `test_sources_cannot_be_changed_for_approved_versions` because the previous value belonged to a different user.